feat(docs): add athletex blog post (#1094)
questdb/questdb.io: blog/2022-10-03-athletex-interview.mdx

---

> title: "DevStory #1: Time series for sports prediction market"

Reviewer: ```suggestion
title: "DevStory #1: Time series for sports prediction markets"
```
I searched the terms a bit and wonder if it should be plural?

author: Pei-Shan Wu
author_title: QuestDB Team
author_url: https://github.com/pswu11
author_image_url: https://avatars.githubusercontent.com/pswu11
description:
  This is a brand-new series for which we interviewed different developers in

>   our community. For the post of this series, we interviewed Kevin Kamto,

Reviewer: ```suggestion
  our community. For the first post of this series, we are thrilled to interview Kevin Kamto,
```

>   our community. For the post of this series, we interviewed Kevin Kamto,

Reviewer: I thought I commented already but is "for the post" correct?

Author: @amyshwang it's quite weird cuz it's always been "for the first post"...  even in the preview.

Reviewer: I think these lines are not really rendered in the webpage. I noted it as I ready through the raw file.

  Co-founder at AthleteX Markets.
keywords: [blockchain, defi, crypto, timeseries, database, mlb, athletex]
tags: [community, story, devstory, iot, crypto]
image: /img/blog/2022-10-03/athletex-trade-view.png
---

import Screenshot from "@theme/Screenshot"

> _This is a brand-new series: developer stories, for which we interviewed

Reviewer: ```suggestion
_This is a brand-new series: developer stories, for which we interviewed
```
I don't feel strongly but I wonder if here it should be "DevStory" too? I guess DevStory is just a short hand in title? So all good, as long as we are consistent.

different developers in our community for their actual experience building with

> QuestDB. For the first post of this series, we interviewed Kevin Kamto,
> Co-founder at [AthleteX Markets](https://www.athletex.io/)._

Reviewer: ```suggestion
QuestDB. For the first post of this series, we are thrilled to interview Kevin Kamto,
Co-founder at [AthleteX Markets](https://www.athletex.io/)._
```

<!--truncate-->

### Hi Kevin, could you tell us more about yourself?

My name is Kevin Kamto. I'm one of the core contributors and co-founders of

> AthleteX Markets. That means that on a day to day basis, I operate as head of
> sales, and a point of coordination for the team.

Reviewer: ```suggestion
AthleteX Markets. That means that on a day to day basis, I operate as the head of
sales, and a point of coordination for the team.
```

### How did you get to know time series databases in the first place?

I've been working in the IoT space as a developer for many years. For a cool
project I worked on, we needed to create a device that measured the WiFi usage
in the company to observe the day to day patterns. That's when I got introduced
to InfluxDB, one of the time series database pioneers at that time.

### Tell us more about AthleteX Markets, what is this project about?

This is the backstory: during the middle of COVID-19, there were a lot of
markets seeing red. So we asked ourselves, what is an investor to do during this

> time? As someone who loves sports like soccer and basketball, I imagine that if
> I could put my money towards an athlete, I could keep it safe.

Reviewer: ```suggestion
time? As someone who loves sports like soccer and basketball, I thought that if
I could put my money towards an athlete, I could keep it safe.
```

:::note

> This blog post is not an investment advise. 😇

Reviewer: Emoji not rendering and:
```suggestion
This blog post is not  investment advice. 😇
```
My thoughts for this:
Advice is not countable - so either a piece of advice, or as the suggestion.

:::

> The idea of AthleteX is that you can invest in the performance of your favourite

Reviewer: ```suggestion
The idea of AthleteX is that you can invest in the performance of your favorite
```

  width={650}
/>

### How is the use case related to time series databases?

Using MLB (baseball) as an example, some in-game metrics such as bats and home
runs are generated every minute or every five minute. We must store all that
information over time. Coming from an IoT background, the problem is not new to
me. So instead of going with something like MySQL or MongoDB, which didn't make
a lot of sense, I knew that a time series database was what we needed.

### Among all the available time series databases, why QuestDB?

QuestDB is faster than its competitors and goes above and beyond what we need it
to handle at the current stage. It's also convenient to use SQL as the query
language. At the moment, we store thousands of time series every minute and both
the ingestion and querying performance has been satisfying. At scale, we
anticipate hundreds of millions of data points and expect QuestDB to handle it
smoothly.

### Could you tell us more about the technical side of your use case and how QuestDB fits in?

QuestDB is the core foundation of our entire backend infrastructure. Due to the
feed-type nature of our application and the need to look up athlete stats very

> fast, we need a database to manage large time series data sets and provide fast
> queries time.

Reviewer: "query time"? Or:
```suggestion
fast, we need a database to manage large time series data sets and provide fast
query performance.
```

---
title: "DevStories #1: Time series for sports prediction markets"
author: Pei-Shan Wu
author_title: QuestDB Team
author_url: https://github.com/pswu11
author_image_url: https://avatars.githubusercontent.com/pswu11
description:
  This is a brand-new series for which we interviewed different developers in
  our community. For the post of this series, we interviewed Kevin Kamto,
  Co-founder at AthleteX Markets.
keywords: [blockchain, defi, crypto, timeseries, database, mlb, athletex]

> tags: [community, story, devstory, iot, crypto]

Reviewer: Why is this relevant to iot? It'd be a bit of a stretch to say so?

to InfluxDB, one of the time series database pioneers at that time.

### Tell us more about AthleteX Markets, what is this project about?

This is the backstory: during the middle of COVID-19, there were a lot of
markets seeing red. So we asked ourselves, what is an investor to do during this
time? As someone who loves sports like soccer and basketball, I thought that if
I could put my money towards an athlete, I could keep it safe.

:::note

This blog post is not investment advice. 😇

:::

The idea of AthleteX is that you can invest in the performance of your favorite
athletes. Essentially, we created a fantasy sports prediction market. You can go
to the platform to either long or short the performance of an athlete by trading
the Athlete Performance Tokens (APTs). The price of an APT is based on an
athlete's in-game statistics. Taking baseball as an example, we determine price
using the Wins Above Replacement (WAR) formula, and the price is updated in
real-time.

<Screenshot
  alt="the dashboard reflecting real-time athlete performance and the prices
associated"
  title="The dashboard reflecting real-time athlete performance and the prices

> associated"

Reviewer: No full stop - I think it is fine as this is not a completed sentence. But the caption for other images has a full stop...

import Screenshot from "@theme/Screenshot"

> _This is a brand-new series: developr stories, for which we interviewed

Reviewer: developer?

developers in our community for their actual experience building with QuestDB.

> For the first post of this series, we are trilled to interview Kevin Kamto,

Reviewer: ```suggestion
For the first post of this series, we are thrilled to interview Kevin Kamto,
```

### Anything more you'd like to tell the readers?

> AtheleteX 1.0.0 has just been released on Sept 10, 2022. If you're interested
> learning more about the project, check out

Reviewer: ```suggestion
AtheleteX 1.0.0 has just been released on Sept 10, 2022. If you're interested
in learning more about the project, check out
```
